spec/index.bs

```diff
@@ -12840,6 +12842,20 @@ GPUQueue includes GPUObjectBase;
                     </div>
                 1. Issue: Do the actual copy.
             </div>
+            <div class=note heading>
+                When copying from a `"webgl"`/`"webgl2"` context canvas, the
+                [=WebGL Drawing Buffer=] may be not exist during certain points in the
+                frame presentation cycle (after the image has been moved to the compositor
+                for display). To avoid this, either:
+
+                - Issue {{GPUQueue/copyExternalImageToTexture()}} in the same [=task=] with
+                    WebGL rendering operation, to ensure the copy occurs before the WebGL
+                    canvas is presented.
+                - If not possible, set the `preserveDrawingBuffer` option in
+                    {{WebGLContextAttributes}} to `true`, so that the drawing buffer will
+                    still contain a copy of the frame contents after they've been presented.
+                    Note, this extra copy may have a performance cost.
+            </div>
```

Comment:
Actually I noticed you considered different places to put this - I'm going to move it up closer to the prose documentation since it's targeted at developers (whereas the algorithm definitions are more targeted at implementers)